cmd/send.go

```diff
@@ -56,7 +59,15 @@ var sendcmd = &cobra.Command{
 				ExitWithError(err.Error())
 			}
 			if statusBar.success {
-				fmt.Println("Send tokens success")
+				if doJSON {
+					j := map[string]string {
+		  			"status": "success",
+		  			"tx": txn.Hash(),
+		  			"nonce": strconv.FormatInt(txn.GetTransactionNonce(),10) }
+					util.PrintJSON(j)
+					return
+				}
+				fmt.Println("Send tokens success:", txn.Hash())
```

Comment:
revert it to fix system_test.
```suggestion
fmt.Println("Send tokens success:")
```